production/technical_userguide.md

```diff
@@ -67,15 +65,44 @@ Client Secret:
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-In the following cURL command, notice that we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which requires base-64 encoding be performed on your clientId and secret**
+
+Format:
 
 ```
-curl -d '' -X POST "https://api.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic MDk4NjlhN2YtNDZjZS00OTA4LWE5MTQtNjEyOWQwODBhMmFlOjY0OTE2ZmU5NmY3MWFkYzc5YzU3MzVlNDlmNGU3MmYxOGZmOTQxZDBkZDYyY2Y0M2VlMWFlMDg1N2UyMDRmMTczYmExMGU0MjUwYzEyYzQ4"
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
```

Comment:
To me, the `Format` section seems a bit confusing, because it shows an incomplete `curl` command.  

What if, instead of showing `curl` in the `Format` section, we simply describe what should be done for Basic Auth as part of this option?  That way, the incomplete `curl` in the `Format` section doesn't confuse folks looking at the real `curl` in the `Example` section.

So, for this one (option 1) we could say something like (please make this sound better:
```
With this option, it is required that the user base64 encode the `clientId:secret`.  Once that activity is performed, the encoded credentials can be passed to `curl` as a header with the form: `authorization: Basic [base64-encoded clientId:secret]`
```

Reply:
Accepted, resolved.